ddl/placement_rule_test.go

```diff
@@ -194,26 +194,11 @@ func (s *testPlacementSuite) TestPlacementBuild(c *C) {
 		{
 			input: []*ast.PlacementSpec{
 				{
-					Role:        ast.PlacementRoleLearner,
-					Tp:          ast.PlacementAdd,
-					Replicas:    3,
-					Constraints: `["+  zone=sh", "-zone = bj"]`,
-				},
-				{
-					Role: ast.PlacementRoleVoter,
+					Role: ast.PlacementRoleLearner,
 					Tp:   ast.PlacementDrop,
 				},
 			},
-			output: []*placement.Rule{
-				{
-					Role:  placement.Learner,
-					Count: 3,
-					LabelConstraints: []placement.LabelConstraint{
-						{Key: "zone", Op: "in", Values: []string{"sh"}},
-						{Key: "zone", Op: "notIn", Values: []string{"bj"}},
-					},
-				},
-			},
```

Comment:
Why discard the previous cases?

Reply:
It is no longer a valid case. The original case is to prove `drop voter` will not effect learner. It will silently succeed, which is just oppsite to what I want to do now.

I have added a new similar case.